libpod/container_api.go

```diff
@@ -109,7 +109,7 @@ func (c *Container) Start(ctx context.Context, recursive bool) error {
 // Either resources, restartPolicy or changedHealthCheckConfiguration must not be nil.
 // If restartRetries is not nil, restartPolicy must be set and must be "on-failure".
 // Nil values of changedHealthCheckConfiguration are not updated.
-func (c *Container) Update(resources *spec.LinuxResources, restartPolicy *string, restartRetries *uint, changedHealthCheckConfiguration *define.UpdateHealthCheckConfig) error {
+func (c *Container) Update(resources *spec.LinuxResources, restartPolicy *string, restartRetries *uint, changedHealthCheckConfiguration *define.UpdateHealthCheckConfig, env *[]string, unsetEnv *[]string) error {
```

Comment:
At this point, should refactor so it accepts a struct instead of a half-dozen arguments. Libpod API isn't considered stable, so this won't be a breaking change.

Reply:
Thanks, I will do it in the next update tomorrow :-).

Reply:
I've changed the code so the ContainerUpdateOptions is passed to Update method now instead of this list of args.